index.bs

```diff
@@ -1372,6 +1378,7 @@ that are returned to the caller when a new credential is created, or a new asser
         [SameObject] readonly attribute AuthenticatorResponse    response;
         [SameObject] readonly attribute DOMString?               authenticatorAttachment;
         AuthenticationExtensionsClientOutputs getClientExtensionResults();
+        static boolean isConditionalMediationAvailable();
```

Comment:
`isUserVerifyingPlatformAuthenticatorAvailable` returns a `Promise<Boolean>`, should we continue that pattern here? (Question, not a proposal)

Reply:
Good question. IIRC @nsatragno has some reasoning as to why it is not cast as a Promise. I _think_ it might be that this method is a query for the client itself and can be instantly answered without having to, for example, assess the platform's capabilities (which might take some time).

Reply:
Agreed, sounds good to me.